From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: "Jan Alexander Steffens (heftig)" <heftig@archlinux.org>
Date: Wed, 12 Jul 2023 00:10:11 +0200
Subject: [PATCH] Use C++17 for protobuf tests

Our protobuf (and abseil-cpp) can currently only be used with C++17.
---
 test cases/frameworks/5 protocol buffers/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test cases/frameworks/5 protocol buffers/meson.build b/test cases/frameworks/5 protocol buffers/meson.build
index 9a4154b652f3..f9e677c95f8f 100644
--- a/test cases/frameworks/5 protocol buffers/meson.build	
+++ b/test cases/frameworks/5 protocol buffers/meson.build	
@@ -1,4 +1,4 @@
-project('protocol buffer test', 'cpp', default_options: ['cpp_std=c++14'])
+project('protocol buffer test', 'cpp', default_options: ['cpp_std=c++17'])
 
 protoc = find_program('protoc', required : false)
 dep = dependency('protobuf', required : false)
